test/catch-override-exception.test.js

```diff
@@ -1,10 +1,10 @@
 'use strict'
 
-const { test } = require('tap')
+const { test } = require('node:test')
 const boot = require('..')
 
-test('catch exceptions in parent.override', (t) => {
-  t.plan(2)
+test('catch exceptions in parent.override', (t, testCompleted) => {
```

Comment:
Should be called done

Reply:
It is used to prevent mixing up with the avvio `done` callback.
https://github.com/fastify/avvio/pull/277#discussion_r1898357599

Reply:
as suggest by @Eomm  in https://github.com/fastify/avvio/pull/277, i think we should keep the name of second variable consistent

Reply:
@Eomm had an issue with the fact it was called `done_` I guess, it should have been called `testDone`.

Reply:
It's ok to use `testDone` or `testCompleted`,  we should keep consistent in all tests.
If use `testDone`,  i would replace `testCompleted` with `testDone` in `basic.test.js`  and `callbacks.test.js`